deploy/test/test_cases/TEST_ID_23_helm_validate_K8S_SECRETS_env_var_incorrect_value.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -euxo pipefail
+
+# This test verifies that the Secrets Provider mechanism works when deploying via HELM and the proper logs are returned
+# TODO verify Job start up and completed logs
+
+setup_helm_environment
+
+pushd ../../../
+  # Install HELM with a K8s Secret that does not exist
+  export K8S_SECRETS="K8S_SECRET-non-existent-secret"
+  export LABELS="app: test-helm"
```

Comment:
Why we need this ?

Reply:
as we discussed, this is just a description on the pod